x-pack/plugins/maps/public/kibana_services.ts

```diff
@@ -75,8 +74,22 @@ export const getEMSSettings = () => {
 
 export const getEmsTileLayerId = () => getKibanaCommonConfig().emsTileLayerId;
 
-export const getRegionmapLayers = () => _.get(getKibanaCommonConfig(), 'regionmap.layers', []);
-export const getTilemap = () => _.get(getKibanaCommonConfig(), 'tilemap', []);
+export const getRegionmapLayers = () => {
+  const config = getKibanaCommonConfig();
+  if (config.regionmap && config.regionmap.layers) {
+    return config.regionmap.layers;
+  } else {
+    return [];
+  }
+};
+export const getTilemap = () => {
+  const config = getKibanaCommonConfig();
+  if (config.tilemap) {
+    return config.tilemap;
+  } else {
+    return {};
```

Comment:
Not sure if this was intentional or not but returning `{}` while original returned `[]`

Reply:
the `[]` was wrong. probably a copy paste from the original `_`.